service.go

```diff
@@ -97,7 +95,11 @@ func (s *Service) HasAlias(alias string) bool {
 }
 
 func (s *Service) HasTag(key string, value string) bool {
-	for _, tag := range s.Tags.Nodes {
+	tags, err := s.Tags(NewGQLClient(), nil)
```

Comment:
😬 we cannot do this.  This might not work depending on what kinds of configuration has happened.

This method should be moved to TagConnection

Reply:
🧐 Yes I see... this would _clearly_ result in chaos 😅